contracts/contracts/TACoApplication.sol

```diff
@@ -611,7 +618,7 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
     /**
      * @notice Returns staking provider for specified operator
      */
-    function stakingProviderFromOperator(address _operator) external view returns (address) {
+    function stakingProviderToOperator(address _operator) external view returns (address) {
```

Comment:
This should be operatorToStakingProvider, isn't?

Reply:
of course, my bad

Reply:
✔️